components/bl-leafletMap-component/README.md

```diff
@@ -62,3 +62,83 @@ The component covers all the main features of the library and allows you in a co
 @bl-customComponent-leafletMap-height: 400px;
 @bl-customComponent-leafletMap-margin: 0;
 ````
+## Examples
```

Comment:
Rename to Codeless Examples and rework as we discussed

Short description -> Image

Example:

```
## Codeless Examples

Addition of markers on map:

<image>
```